calcom/cal.com
# refactor: Update test utils

## What does this PR do?

- One helper function was added for asserting toaster text. None of the tests has toaster text assertion. This function could help in all the specs where toaster text assertion could be included. Added one check-in team's creation test.
- Also, one extra assertion was added in the team's spec to verify slug remains the same after the team created

## Type of change

- Improvement in test utility library


## How should this be tested?

Run teams.e2e.ts test multiple times and see test passes always


I removed my old PR since I added yarn.lock file. This time, I have not committed yarn.lock file. 

@kemiller Please review as per closed PR https://github.com/calcom/cal.com/pull/9390
